docs/getting-started-guides/docker-multinode.md

```diff
@@ -0,0 +1,261 @@
+### Running Multi-Node Kubernetes Using Docker
+
+_Note_: These instructions are somewhat significantly more advanced than the [single node](docker.md) instructions.  If you are
+interested in just starting to explore Kubernetes, we recommend that you start there.
+
+
```

Comment:
At the end of this guide you will have a setup that supports adding additional worker nodes after the initial bootstrapping process.